lib/reconcile/cassandra.go

```diff
@@ -268,3 +275,54 @@ func getCassandraNodes(cr *apiv1alpha1.Astarte) string {
 
 	return strings.Join(nodes, ",")
 }
+
+func handleCassandraUserCredentialsSecret(statefulSetName string, cr *apiv1alpha1.Astarte, c client.Client, scheme *runtime.Scheme) error {
```

Comment:
As the linter pointed out, you should make the RMQ function generic, and invoke it for both RMQ and Cassandra